torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -220,6 +159,7 @@ void ProfilingRecord::insertShapeProfile(Node* n, size_t offset) {
 }
 
 bool needsProfiledInputs(Node* n) {
+  // Add CUDAFuser checks
```

Comment:
@jjsjann123 I couldn't find the equivalent of `tensorexpr::isSupported`

Reply:
Here you go:
https://github.com/csarofeen/pytorch/blob/sum_to_size_PR/torch/csrc/jit/codegen/cuda/register_interface.cpp#L27-L28

`isSupported` should be `canFuseNode`, additional for ivalue profiling, we have `insertProfileIValue`